spec/features/01_visitor_sees_fruit_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature "visitor sees content" do
+  scenario "fruits are on the page" do
+
+    banana = Fruit.create!(name: "banana", description: "yellow", sci_name: "plantae")
+    visit fruits_path
+
+    expect(page).to have_content("Fruits")
+    expect(page).to have_content banana.name
+    expect(page).to have_content banana.sci_name
+  end
+end
```

Comment:
for an index test, ideally we test that at least two fruits are being tested for.

Reply:
added another fruit